bin/metababel

```diff
@@ -87,30 +99,51 @@ end
 # Indeeed the <%= body %> will be indented, but we don't don't want it,
 # in the body string is already indented
 # But we clean the white space empty line afterward \o/
-def wrote_event_dispatchers(folder, t)
+def wrote_event_dispatchers(folder, t, m, callback_types)
   event_name = 'event'
+
+  matching_dispatchers = m.map do |md|
+    event, clock = t.find_event_class { |e| Regexp.new(md[:regex]).match?(e.name) }
+    raise "Matching callback #{md[:name]} does not match any event in model." unless event
```

Comment:
Maybe more of a warning? No strong opinion...

Reply:
Do we want to render datatypes for a matching callbacks that won't be used?